babel-watch.js

```diff
@@ -156,12 +157,13 @@ const watcher = chokidar.watch(program.watch, {
 let watcherInitialized = (program.watch.length === 0);
 debugInit('Initializing babel-watch with options: %j', program.opts());
 
-process.on('SIGINT', function() {
-  debugInit('SIGINT caught, closing.');
+
+onExit(function(code, signal) {
+  debugInit(`${signal} caught, closing.`);
   watcher.close();
   killApp();
-  process.exit(0);
-});
+  process.exit(code || 0);
```

Comment:
Sending SIGINT, SIGTERM and possibly any other signal, gives `code: null`
Running process.exit(code), results into onExit called with the given code value